default_settings.lua

```diff
@@ -48,7 +48,7 @@ pipeworks.toggles = {}
 -- setting to nil inhibits all flow logic, useful for debugging ABM crashes,
 -- or for rendering the pipes purely decorative.
 ]]
-pipeworks.toggles.pipe_mode = "classic"
+pipeworks.toggles.pipe_mode = "pressure"
```

Comment:
how does that affect existing worlds?

Reply:
a lot of the things I'm doing *are* going to break things, but the old way things work is likely simple enough to auto-reinstate the original function